modules/installation-configuration-parameters.adoc

```diff
@@ -58,6 +54,7 @@
 // * installing/installing-restricted-networks-nutanix-installer-provisioned.adoc
 // * installing/installing_vsphere/installation-config-parameters-vsphere.adoc
 // * installing/installing_gcp/installation-config-parameters-gcp.adoc
+// * installing/installing_gcp/installation-config-parameters-ibm-z.adoc
```

Comment:
@SNiemann15 Please update the Where used list to reflect that this module now appears in the second assembly you created.